rewrite_rules/rewrite_geometry_input.py

```diff
@@ -0,0 +1,28 @@
+import re
+from typing import Dict, List, Tuple
+
+from rewrite_rules.rewrite_base import RewriteBase
+from vprint import vprint1
+
+
+class GeometryInput(RewriteBase):
+    rgx_input = re.compile(r"geometry_input\s*\(\s*([^\s][^,\s]*[^\s,)])\s*([,]?[^\)]*)\s*\)")
+    rgx_output = re.compile(r"geometry_output\s*\(\s*([^\s][^\)]*[^\s])\s*(\))")
```

Comment:
Is `[^\<single character or range>]` really what you need? it will also match unprintable characters and other special characters like `~%{}[]^@#&+- etc...`. imo `\w\s_=` should be all you need

Reply:
not sure am still learning regex, I haven't come across `\w\s_=` yet